inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/convolution/convolution_kernel_ref.cpp

```diff
@@ -13,6 +13,7 @@ ParamsKey ConvolutionKernel_Ref::GetSupportedKey() const {
     ParamsKey k;
     k.EnableInputDataType(Datatype::F16);
     k.EnableInputDataType(Datatype::F32);
+    k.EnableInputDataType(Datatype::INT32);
```

Comment:
I also think we can add UINT32 input and INT32/UINT32 output, too, cause this is the reference code. @vladimir-paramuzov how do you think?

Reply:
I think it makes sense to add it if kernel really supports (U)INT32 output